olp-cpp-sdk-dataservice-write/test/unit/TestVersionedLayerClient.cpp

```diff
@@ -78,16 +78,19 @@ TEST_P(VersionedLayerClientOfflineTest, StartBatchRequestTest) {
 }
 
 class VersionedLayerClientOnlineTest : public VersionedLayerClientTest {
- public:
  protected:
+  static std::shared_ptr<olp::http::Network> s_network;
```

Comment:
```suggestion
  // Static network instance is necessary as it needs to outlive any created clients.
  // This is a known limitation as triggered send requests capture the network instance inside the callbacks.
  static std::shared_ptr<olp::http::Network> s_network;
  
```

Add newline before `SetUpTestSuite()`